### PR TITLE
🚐 Chrysler and Jeep support for OpenPilot

### PR DESCRIPTION
**Pull request by adhintz**
Uses OpenPilot for steering and the car's built-in Adaptive Cruise Control (ACC) for acceleration and braking. To engage OpenPilot, engage the car's ACC. To disengage OpenPilot, press the ACC cancel button, press the brake, or press the gas pedal.

Steers down to 7mph, but currently set to 9mph minimum steering speed to avoid faulting the steering actuator for that drive when sharply decelerating. ACC is stop-and-go, but if the car is stopped for more than 3 seconds, then it requires the user to press the gas pedal to resume.

This version has been used on 9 different cars. Longest drive I've done without taking control is ~2 hours on I-5.

More details are available here: https://github.com/adhintz/openpilot/wiki 

Big thanks to @itrealizer for the Jeep and 2018 Pacifica non-hybrid fingerprints, recorded drives, and encouragement!

### CODE REVIEW

**Comment by rbiasini** [comment]
This is great @adhintz ! We will review and test over the holidays.

File: README.md [hunk 1]
```diff
@@ -140,6 +140,8 @@ We've written guides for [Brand](https://medium.com/@comma_ai/how-to-write-a-car
 - We put time into a Ford port, but the steering has a 10 second cutout limitation that makes it unusable.
 - The 2016-2017 Honda Accord uses a custom signaling protocol for steering that's unlikely to ever be upstreamed.
 
+- Chrysler Pacifica Hybrid 2017
```
Comment (rbiasini) [inline README.md]: this should go in the main table

Comment (adhintz) [reply README.md]: Done.

**Comment by rbiasini** [comment]
Chrysler support present in 0.5.8. Nice work @adhintz , this was a tough port.

**Pull request closed without merging**